### PR TITLE
[3D] mouse position

### PR DESCRIPTION
**Pull request by fredj**

### CODE REVIEW

**Comment by fredj** [comment]
demo: https://mf-geoadmin3.dev.bgdi.ch/mouseposition_3d/?dev3d=true

**Comment by cedricmoullet** [comment]
When I come back in 2D, the mouse position tool is not activated

**Comment by gjn** [comment]
I can confirm. The re-enabling of 2D Map coordinates is not working correctly.

File: src/components/mouseposition/MousePositionDirective.js [hunk 1]
```diff
@@ -23,6 +25,45 @@ goog.provide('ga_mouseposition_directive');
...
         });
+
+
+        scope.$watch('::ol3d', function(ol3d) {
+          if (ol3d) {
+            var scene = ol3d.getCesiumScene();
+            var ellipsoid = scene.globe.ellipsoid;
+            var handler = new Cesium.ScreenSpaceEventHandler(scene.canvas);
+
+            handler.setInputAction(function(mouvement) {
```
Comment (gjn) [inline src/components/mouseposition/MousePositionDirective.js]: This is called alot of times when moving the mouse. Maybe we should throttle this (we have a `gaThrottle` class for this) to 100ms or 200ms.

**Comment by fredj** [comment]
Issues fixed, please test again

**Comment by cedricmoullet** [comment]
Works fine.
@gjn: could you review and merge if ok ?

**Pull request merged**